packages/nimble-blazor/Tests/NimbleBlazor.Tests.Acceptance/Tests/WaferMapTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Playwright;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Acceptance
+{
+    public class WaferMapTests : AcceptanceTestsBase
+    {
+        public WaferMapTests(PlaywrightFixture playwrightFixture, BlazorServerWebHostFixture blazorServerClassFixture)
+            : base(playwrightFixture, blazorServerClassFixture)
+        {
+        }
+
+        [Fact]
+        public async Task WaferMap_RendersCircleAsync()
```

Comment:
@atmgrifter00 I might be missing it but doesn't look like we have acceptance tests that exercise all the JS in NimbleBlazor.lib.module.js. The ones that come to mind are the getValidity methods. Did I just miss it? Is it known we have that gap and is it tracked anywhere? If not wanna make an issue to track it :)

Reply:
This is a known gap, and I'm not sure we can address it. I don't believe we can properly assert that we've executed JS code if we can't access actual DOM elements. All Playwright gives us (afaik) is their `ILocator` objects which are an abstraction around the elements, and thus don't give us access to the mechanisms we need to assert as desired. If what we were asserting against was DOM state (like attributes) the locator APIs would be sufficient. @msmithNI can correct me here if I'm mistaken.

Reply:
This is definitely a gap, but I was unaware of how to bridge it, and I'll admit I'm still not sure how the `evalutate/evaluateHandle` APIs can accomplish it, but maybe they do? It's definitely worth tracking in an issue.

Reply:
The evaluate/evaluateHandle methods weren't what I had in mind for this case. We want to see that the actual method execution and Blazor <-> JS communication is working. I think the example added in this PR where a button is pressed programmatically in playwright and a result is displayed somewhere (like old school LV boolean assert tests) would do the trick. Captured notes in: https://github.com/ni/nimble/issues/1804